radis/misc/profiler.py

```diff
@@ -40,8 +41,10 @@ def __init__(self, verbose):
 
         # Dev: Init here to be found by autocomplete
         self.initial = {}
-        self.dict_time = {}
         self.verbose = verbose
+        self.current_verbose2_counter = ""  # Holds current Verbose level 2 key
+        self.final = OrderedDict()
+        self.flag = False  # For checking transit between verbose level 3 to 2
```

Comment:
@anandxkumar I merged to test on my own PR #325 , and I got it working. 

However, this part of the code here looks a bit complicated : why do you need a special "flag" for difference level 2 and 3 ? What if there's level 4, or 5 ?  Would it still work ?

Reply:
No in that case it shouldn't work, because I wrote it assuming we are dealing with 3 levels. The reason I put a flag here is to see if we are transit from level 3 to level 2 when using `profiler.stop`. If yes then we append "value". But a better approach will be to check the len of that key in the dictionary. if `length` != 0 then we add it in the form 'value': val. This will remove the need of `self.flag`. In order to generalize it I will need to do some modifications adding the key in the dictionary so it will require keeping track of the all the previous levels latest key to add the current key in it (self.final['key1']['key2']['key3'].update({'currentkey':{}}) and yet this approach will rely on the number of levels we have overall.